app/services/rfd.local.server.ts

```diff
@@ -88,3 +105,9 @@ export function fetchLocalRfds(): LocalRfd[] {
 
   return rfds
 }
+
+function checkForRelativePath(path: string) {
+  if (path.includes('..')) {
+    throw new Error('Path must not include any relative path parts')
```

Comment:
I'd log the bad path here and say "include path", not just path, to make it easier to figure out the problem